database/cassandra/cassandra_test.go

```diff
@@ -11,8 +11,8 @@ import (
 )
 
 var versions = []mt.Version{
-	{"cassandra:3.0.10", []string{}},
-	{"cassandra:3.0", []string{}},
+	{Image: "cassandra:3.0.10"},
```

Comment:
Switched to named struct values so that the Cmd doesn't need to be specified (see below).